contributors/guide/scalability-good-practices.md

```diff
@@ -16,42 +16,38 @@
 - [Summary](#summary)
 - [Closing remarks](#closing-remarks)
 
-## Who should read this document and what's in it?
-This document is targeted at developers of "vanilla Kubernetes" who don't want their changes rolled-back or blocked because of performance regressions they cause. It contains some of the knowledge and experience gathered by the scalability team in over two years.
+## Who should read this document and what is in it?
+This document is targeted at developers of "vanilla Kubernetes" who do not want their changes rolled-back or blocked because they cause performance regressions. It contains some of the knowledge and experience gathered by the scalability team over more than two years.
  
-It is presented as a set of real, or close-to-real examples from the past that broke scalability tests followed by some explanations and general suggestions on how to avoid causing similar problems.
+It is presented as a set of past examples, as realistic as possible, which broke scalability tests. Those are followed by explanations and general suggestions on how to avoid causing similar problems.
 
 ## TL;DR:
-1. You should understand how go memory management work to write code that is fast and doesn't use too much memory - pretty obvious, but we spent noticeable amount of time removing various bad patterns. In particular:
-pass arguments by pointers wherever it's correct
+1. To write fast code with efficient memory management, you must understand how the go language manages memory. This would seem obvious, but we have spent a considerable amount of time removing various bad patterns. In particular:
+wherever it is correct, pass arguments by pointers
 avoid unnecessary copying of data (especially slices and maps)
```

Comment:
I replaced the dash with a comma when I fixed the run-on sentence.

Reply:
This will render as a single paragraph in markdown as is. Either make a bulleted list, or re-work lides 26, 27, and 28 into a full sentence with punctuation.

Reply:
See my comment below - look at the rendered markdown output; you will see it just smashes lines 15-28 together with no newlines.

Reply:
Made bulleted list.